src/main/java/edu/nextstep/camp/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,50 @@
+package edu.nextstep.camp.calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String nums) {
+        if (!validateEmptyNums(nums)) {
```

Comment:
```suggestion
        if (isEmptyNums(nums)) {
```
이런 식의 함수명은 어떨까요?
boolean 타입을 반환하는 함수명은 is~~ 로 명명했을 때 가독성을 향상시켜 주는 것 같아요